src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  margin: 0 auto;
```

Comment:
It's redundant as the width is 100% anyway and top and bottom margins are zero by default.